src/test/java/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,32 @@
+package duke.task;
+
+import duke.util.DateAndTimeFormatter;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Might be better to add an empty line to separate import statements from different dependencies?